src/features/common/types/projectv2.d.ts

```diff
@@ -15,3 +17,19 @@ export type MapProjectProperties =
 export type ExtendedProject = TreeProjectExtended | ConservationProjectExtended;
 
 export type MapProject = ProjectMapInfo<MapProjectProperties>;
+
+export interface Image {
+  image: string;
+  description: Nullable<string>;
+  id: string;
+}
+
+export type SliderImage = {
+  image?: string | undefined;
+  description?: string | null;
+};
+
+export type PointPlantLocation =
+  | PlantLocationSingle
+  | SamplePlantLocation
+  | null;
```

Comment:
1. Adding null to the type is confusing. 
2. Personally, I prefer just using the constituent types directly, since this is only used in 2 places, and the type only has 2 parts (excluding the null).